spec/lib/bip_claims/service_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'rails_helper'
 require 'bip_claims/service'
+require 'mpi/attr_service'
```

Comment:
This should actually go in `lib/bip_claims/service.rb`

Reply:
👍